Gemfile

```diff
@@ -7,4 +7,5 @@ gem 'rubocop', '~> 0.60.0', require: false
 group :test, :remote_test do
   # gateway-specific dependencies, keeping these gems out of the gemspec
   gem 'braintree', '>= 2.93.0'
+  gem 'mechanize'
```

Comment:
We need mechanize so that we can complete the Stripe 3DS 1.0 authentication step for purchases. This is done through URL provide by Stripe and requires a button to be clicked to change the 3DS source from pending to chargeable.